### DISCUSSION TITLE
Is there support for openhab Cloud?

### DISCUSSION THREAD

**Discussion by wollyd** [seaside1/jrule#162] [Q&A]
Hi,

is openhab Cloud  supported ?
And if so how to send a notification to it ?

kind regards

willy

**Comment by querdenker2k** [comment]
Hi, 
I don't use openhab cloud. Isn't it just for a remote access to your local instance?
If so it should be supported. Notifications could be possible if this is done via actions. Check out JRuleActions for this.

Kind regards

**Reply by wollyd** [reply]
hi,
If I look in the JRuleActions.java, there is no definition of a remote openhab cloud. 
As far as I see it , remote openhab cloud is not really a binding, in the openhab GUI it is configured in the settings and does not show up in the things.
kind regards

**Comment by querdenker2k** [comment]
Sorry, but i am not using remote openhab @seaside1 are you using it?

**Comment by seaside1** [comment]
No I'm not. I did use it at an early stage, but tend to use VPN to access my local openHAB instance now. Out of interest what is you want to achieve?  JRule works within openHAB triggering on events that takes place on the eventbus. If you have an Item you can access/ or change via openHAB cloud that should not affect rules within openHAB.
Would be easier if you could describe the scenario with some more details.

Best Regards, S

**Reply by wollyd** [reply]
I just want to send a notification to my app on my cell (see DSL rules sendNotification)

kind regards
Willy

**Comment by querdenker2k** [comment]
@seaside1 Do you know why it's not just working to import openhabcloud (as provided) and use NotificationAction directly? The cloudService field inside of NotificationAction isn't set and therefor nothing is sent.

**Reply by querdenker2k** [reply]
got it, found the ActionClassLoader which does the trick.

**Reply by wollyd** [reply]
@querdenker2k , could you please show an example how to do this ?
Thanks in advance

**Reply by Bruetti1991** [reply]
@querdenker2k Could you please explain how you did it?
